src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.tsx

```diff
@@ -161,7 +166,10 @@ const GeneSummary = () => {
             <div className={styles.downloadWrapper}>
               <InstantDownloadGene
                 genomeId={ensObjectGene.genome_id}
-                gene={{ id: gene.stable_id, so_term: gene.so_term }}
+                gene={{
+                  id: gene.stable_id,
+                  biotype: gene.metadata.biotype.value as string
```

Comment:
I think we made a mistake in the declaration of our metadata types, the results of which we are now seeing here.

Our types should know that `gene.metadata.biotype.value` (or `transcript.metadata.biotype.value`) is a string.

Compare: this is currently our situation (you can see a popup above the `valueField` on the screenshots below with the type inferred by typescript):

<img src="https://user-images.githubusercontent.com/6834224/126367764-7554d0cb-8c5c-45bc-9a33-39ed715254d1.png" width="400" />

as opposed to what it should be instead:

<img src="https://user-images.githubusercontent.com/6834224/126367894-c696d8e2-5d19-48d0-8943-95f5a6b75c32.png" width="600" />


Link to [typescript playground](https://www.typescriptlang.org/play?#code/C4TwDgpgBAaghgGwK4QMoWAWQ3AJnYOKAXigG8AoKKAN0RQC4oBnYAJwEsA7AcygB8oXJAFsARhDYCoYgPayEEOFwDcVKAjgSETVp15rquCADNuHYB1ldd7bjzUBfNRVCQoAIStuI2QvkISKAB5EQsAHnhkNAw-PAI4ABooAHI6aJSAPigAMnJaeghbfT5HCgpFYBlvcCLYQvQsHAC4FwqMAuiAMQ4IBFwmH1kTatkfAG00wpSAXS).

Reply:
valueField still showing all three types for me
![Screenshot 2021-07-20 at 18 43 43](https://user-images.githubusercontent.com/6347854/126370883-b52f8b88-7cb5-412a-a7db-455377853019.png)

Reply:
Now change the `biotype`'s type to BiotypeMetadata in the playground :smiley:

Reply:
got it.. thanks

Reply:
can't we just write `type BiotypeMetadata = ValueSetMetadata & { value: string };`

Reply:
> This will override the current type of value.

Interesting. I thought that it wouldn't and that the more broad `string | number | boolean` would just swallow the more narrow `string`. But you are right, it seems to work, and it looks better than the one with Omit.

Reply:
Cool. Can you also tell me which of the metadata field we expect can have other types (numbers/boolean) for the value attribute?

Reply:
Well, transcript canonical metadata has a value that is a boolean. Not that it's very important, because if a transcript is not canonical, the whole `metadata.canonical` field will be set to null. But if a transcript is canonical, then the `metadata.canonical.value` field will be `true`.

Reply:
ok cool. Means I can set the default type for value as string and override canonical with boolean.